cmd-car-split.go

```diff
@@ -53,6 +57,12 @@ type subsetInfo struct {
 	blockLinks []datamodel.Link
 }
 
+type carFile struct {
+	name       string
+	commP      cid.Cid
+	paddedSize uint64
```

Comment:
I thin k it would also be useful to have the actual file size, too

Reply:
the padded size will be the actual file size, no?